doc/classes/RenderingServer.xml

```diff
@@ -4666,6 +4676,24 @@
 		<constant name="VIEWPORT_MSAA_MAX" value="4" enum="ViewportMSAA">
 			Represents the size of the [enum ViewportMSAA] enum.
 		</constant>
+		<constant name="VIEWPORT_ANISOTROPY_DISABLED" value="0" enum="ViewportAnisotropicFiltering">
+			Anisotropic filtering is disabled.
+		</constant>
+		<constant name="VIEWPORT_ANISOTROPY_2X" value="1" enum="ViewportAnisotropicFiltering">
+			Use 2× Anisotropic Filtering.
```

Comment:
While I did mention that it's best if "Filtering" is lowercase, I was not specific enough and did not mention that it should be like all lowercase for all **ViewportAnisotropicFiltering** and **AnisotropicFiltering** enum values. Sorry
```suggestion
			Use 2× anisotropic filtering.
```